roles/simplebook/defaults/main.yml

```diff
@@ -0,0 +1,2 @@
+---
+simplebook_git_version: main
```

Comment:
I have gone back and forth on whether we should tag the git_versions to a moving target.  But at least for production, and likely demo (in the prod/group_vars/all), we should set to a version.  That way we know, for a given deployment of a version of torque-sites, it corresponds directly to a sha, so re-running the ansible-playbook without updating torque-sites installs the same system, and no one accidentally breaks production by accidentally checking in breaking code to simplebook/main

Reply:
Totally fine.  Though we (really, I) have been using shas instead of tags as that provides more flexibility.  That's not an endorsement, though :)